src/js/angular-tooltips.js

```diff
@@ -77,12 +79,7 @@
 
         thisElement.bind(showTriggers, function onMouseEnterAndMouseOver() {
 
-          if (!initialized) {
-
-            initialized = true;
-            $scope.initTooltip(side);
-          }
-
+          $scope.initTooltip(side);
```

Comment:
@45kb  I also found a bug:
When I hover on the element, tooltip shows. However, if I click the element, (in my case the element will change its position)  the tooltip doesn't change its position as long with the element.

I haven't solved this problem, do you have any idea?